.gitignore

```diff
@@ -1,7 +1,7 @@
-/rust/target/
+target
 result
 .gitconfig
-.direnv/
+.direnv
```

Comment:
Does this final slash not indicate that the ignored thing must be a directory?

Reply:
You're probably right. In that case, shouldn't we have a slash after `result`?

Reply:
> You're probably right. In that case, shouldn't we have a slash after `result`?

Yes, and after 'target'.